src/pages/components/button.mdx

```diff
@@ -58,16 +57,14 @@ See the [Destructive actions pattern](#todo) section for guidelines.
 import ButtonDanger from '../../examples/button/ButtonDanger.tsx';
 import ButtonDangerCode from '!!raw-loader!../../examples/button/ButtonDanger.tsx';
 
-Mark a button disabled with the disabled property.
-[Read more about disabled state usage](#todo)
+A disabled button prevents user interaction and is not perceived by a screen reader.
 
-## Best practices
+## Configuration
 
-### Writing button labels
+<PackageDescription data={props.data.mdx.package} />
```

Comment:
Curious how you feel about the default bullets here? I think @hzhu overrode with a standard `disc` in the markdown mapping.